src/inputs/button/button.jsx

```diff
@@ -127,6 +130,9 @@ const styles = (theme) => {
             },
             '&$text': {
                 color: theme.palette.text.disable,
+                '& .icon .icon-use-path': {
+                    fill: theme.palette.text.primary,
```

Comment:
`theme.palette.text.primary` is deliberate?  Not `theme.palette.text.disable`?

Reply:
I don't remember... checking.

Reply:
Yep, there's a problem, but not on this line, but 132

Reply:
Fixed lined 132.

Reply:
They both match now, so all good.  Resolving.